qiskit_alice_bob_provider/remote/job.py

```diff
@@ -214,12 +215,17 @@ def _ab_event_to_qiskit_status(event: str) -> JobStatus:
         'COMPILATION_FAILED',
         'TRANSPILATION_FAILED',
         'EXECUTION_FAILED',
+        'TIMED_OUT'
     }:
         return JobStatus.ERROR
     elif event == 'SUCCEEDED':
         return JobStatus.DONE
     elif event == 'CANCELLED':
         return JobStatus.CANCELLED
-    raise AliceBobException(
-        f'Received unexpected job event "{event}" from Alice & Bob\'s API'
+    logging.warn(
```

Comment:
Doing this as a drive by so that potential newly introduced events would not make the provider crash